waveorder/stokes.py

```diff
@@ -422,3 +422,37 @@ def mmul(matrix, vector):
         raise ValueError("matrix.shape[1] is not equal to vector.shape[0]")
 
     return torch.einsum("NM...,M...->N...", matrix, vector)
+
+
+def apply_orientation_offset(orientation, rotate, flip):
+    """
+    Applies an in-place rotation and/or flip to an orientation map while
+    keeping the output range within 0 <= orientation < pi.
+
+    Parameters
+    ----------
+    orientation : array_like
```

Comment:
Does this actually work for array-like, e.g. numpy arrays?

Reply:
Thanks, I changed `array_like` to `torch.tensor` everywhere.